build/windows/installer/conf/fluent-bit.conf

```diff
@@ -1,25 +1,61 @@
 [SERVICE]
-    Flush        15
-    Daemon       Off
-    Log_Level    info
-    Log_File      /etc/fluent-bit/fluent-bit.log
+    Flush           15
+    Daemon          Off
+    storage.path  /etc/fluent-bit/flbstore/
+    storage.sync  normal
+    storage.checksum off
+    storage.backlog.mem_limit 10M
+    Log_Level       debug
+    Parsers_File    /etc/fluent-bit/azm-containers-parser.conf
+    Log_File        /etc/fluent-bit/fluent-bit.log
 
 [INPUT]
-    Name            forward
-    Listen          127.0.0.1
-    Port            25230
-    Mem_Buf_Limit   10m
-    Chunk_Size      32
-    Buffer_Size     64
+    Name                tail
+    Tag                 oms.container.log.la.*
+    Path                ${AZMON_LOG_TAIL_PATH}
+    Read_from_Head      true
+    DB                  /etc/fluent-bit/omsagent-fblogs.db
+    DB.Sync             Off
+    Parser              docker
+    ${TAIL_MEM_BUF_LIMIT}
+    ${TAIL_BUFFER_CHUNK_SIZE}
+    ${TAIL_BUFFER_MAX_SIZE}
+    Rotate_Wait         20
+    Refresh_Interval    30
+    Path_Key            filepath
+    Skip_Long_Lines     On
+    Ignore_Older        5m
+    Exclude_Path        ${AZMON_CLUSTER_LOG_TAIL_EXCLUDE_PATH}
 
 [INPUT]
-    Name        tcp
-    Tag         oms.container.perf.telegraf.*
-    Listen      0.0.0.0
-    Port        25229
-    Chunk_Size  32
-    Buffer_Size 64
-    Mem_Buf_Limit 5m
+    Name            tail
+    Tag             oms.container.log.flbplugin.*
+    Path            C:\\var\\log\\containers\\omsagent*.log
+    Read_from_Head  true
+    DB              /etc/fluent-bit/omsagent-fluentbit-containers.db
+    DB.Sync         Off
+    Parser          docker
+    Mem_Buf_Limit   1m
+    Path_Key        filepath
+    Skip_Long_Lines On
+    Ignore_Older    5m
+
+[INPUT]
```

Comment:
why do we need this since we are not forwarding the logs from fluent-d to fluent-bit?

Reply:
ah good point..
is this not needed now as well since we are not forwarding logs: 
https://github.com/microsoft/Docker-Provider/blob/15ee6c53337bec218cabf02aa3234f6b1e0ea412/build/windows/installer/conf/fluent.conf#L81